docs/source/conf.py

```diff
@@ -111,7 +112,7 @@
 
 # List of directories, relative to source directory, that shouldn't be searched
 # for source files.
-exclude_trees = []
+exclude_trees: list[str] = []
```

Comment:
Does mypy really need to run on this file?

Reply:
No, but I also don't mind it. I noticed in a new project that sphinx now removes all unnecessary/optional arguments from `conf.py` when you run the quickstart command. We could consider comparing that to what is in this conf.py and clearing unused things out.